modules/claims_api/config/schemas/v2/2122.json

```diff
@@ -112,20 +112,10 @@
       "type": "object",
       "additionalProperties": false,
       "properties": {
-        "firstName": {
-          "description": "First name of Claimant.",
-          "type": "string",
-          "example": "John"
-        },
-        "middleInitial": {
-          "description": "Middle initial of Claimant.",
-          "type": "string",
-          "example": "M"
-        },
-        "lastName": {
-          "description": "Last name of Claimant.",
+        "claimantId": {
           "type": "string",
-          "example": "Dow"
+          "example": "123456789",
+          "description": "Id of the claimant."
```

Comment:
We should specify that it's an ICN. Or are "id" and "ICN" interchangeable in this context for the API consumers?

Reply:
I was following the pattern we use for veterans elsewhere in Benefits Claims, where we describe the ICN veteranId fields as "Id of the veteran".